src/GraphQl/Resolver/ResourceFieldResolver.php

```diff
@@ -36,8 +36,8 @@ public function __construct(IriConverterInterface $iriConverter)
     public function __invoke($source, $args, $context, ResolveInfo $info)
     {
         $property = null;
-        if ('id' === $info->fieldName && isset($source[ItemNormalizer::ITEM_KEY])) {
```

Comment:
I think it's better to make 'id' and '_id' as consts of `IndentifiersExtractor` class.